catalogue_graph/tests/adapters/axiell/test_trigger.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import annotations
+
+from datetime import UTC, datetime, timedelta
+
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
+from pyiceberg.table import Table as IcebergTable
+
+from adapters.axiell import config
+from adapters.axiell.models.step_events import (
+    AxiellAdapterLoaderEvent,
+    AxiellAdapterTriggerEvent,
+)
+from adapters.axiell.steps import trigger
+from adapters.utils.window_store import IcebergWindowStore, WindowStatusRecord
+
+
+def _window_row(start: datetime, end: datetime) -> WindowStatusRecord:
+    return WindowStatusRecord(
+        window_key=f"{start.isoformat()}_{end.isoformat()}",
+        window_start=start,
+        window_end=end,
+        state="success",
+        attempts=1,
+        last_error=None,
+        record_ids=(),
+        updated_at=end,
+        tags=None,
+    )
+
+
+def _populate_store(
+    table: IcebergTable, rows: list[WindowStatusRecord]
+) -> IcebergWindowStore:
+    store = IcebergWindowStore(table)
+    for row in rows:
+        store.upsert(row)
+    return store
+
+
+def test_build_window_request_uses_lookback_when_no_history(
+    monkeypatch: MonkeyPatch,
+    temporary_window_status_table: IcebergTable,
+) -> None:
+    now = datetime(2025, 11, 17, 12, 0, tzinfo=UTC)
+    monkeypatch.setattr(config, "WINDOW_LOOKBACK_DAYS", 1)
+    store = _populate_store(temporary_window_status_table, [])
+
+    request = trigger.build_window_request(store=store, now=now)
+
+    assert request.window_start == now - timedelta(days=1)
+    assert request.window_end == now
+    assert request.set_spec == config.OAI_SET_SPEC
+    assert request.metadata_prefix == config.OAI_METADATA_PREFIX
+    assert request.job_id
```

Comment:
Is it worth asserting that job_id looks as expected, or is it sufficient that  it is something?